IBAnimatableTests/ColorTypeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ColorTypeTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/26/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
```

Comment:
I'm not really sure of the difference. 
I thought that using **@testable** allows me to test internal methods of IBAnimatable.